assets/src/exit-modal-survey/index.js

```diff
@@ -10,78 +9,76 @@ import './style.css';
 import { data } from '@eventespresso/eejs';
 import { stringify } from 'querystringify';
 
-const { exitModali18n: i18n = {}, exitModalInfo: info = {}} = data;
+const { exitModali18n: i18n = {}, exitModalInfo: info = {} } = data;
 
-const queryString = stringify({
-    firstname: info.firstname,
-    emailaddress: info.emailaddress,
-    website: info.website
-});
+const queryString = stringify( {
+	firstname: info.firstname,
+	emailaddress: info.emailaddress,
+	website: info.website,
+} );
 
 const modalProps = {
-    styles: {
-        content: {
-            display: 'flex',
-            top: '80px',
-            left: '50%',
-            bottom: 'auto',
-            right: 'auto',
-            WebkitTransform: 'translateX(-50%)',
-            transform: 'translateX(-50%)'
-        },
-        typeFormStyle: {
-            width: '600px',
-            height: '400px'
-        }
-    },
-    typeFormUrl: 'https://eventespresso.typeform.com/to/O1DDym?' + queryString,
-    introText: i18n.introText,
-    doSurveyButtonText: i18n.doSurveyButtonText,
-    skipButtonText: i18n.skipButtonText,
-    buttonClass: {
-        doSurvey: 'button button-primary',
-        closeModal: 'button button-secondary'
-    }
+	styles: {
+		content: {
+			display: 'flex',
+			top: '80px',
+			left: '50%',
+			bottom: 'auto',
+			right: 'auto',
+			WebkitTransform: 'translateX(-50%)',
+			transform: 'translateX(-50%)',
+		},
+		typeFormStyle: {
+			width: '600px',
+			height: '400px',
```

Comment:
any  plans to accommodate mobile for this? 
or is this beyond the scope of what I'm  supposed to be reviewing here?